### ISSUE TITLE
saveDatasheet with raster paths

### ISSUE THREAD

**Issue by dmarvs** [ApexRMS/rsyncrosim#2]
I'm running Syncrosim Linux v2.0.9 on a Mac (OSX 10.11), with R v3.3.2. 

When running the R package demo (rsyncrosim-demo.R) everything runs smoothly up until line 165, when I try to save the datasheet containing the absolute paths to the initial spatial conditions. I get the following error:

> Error in tt[[1]] : subscript out of bounds
The specified external file does not exist:
> /Users/davidmarvin/Documents/TNC/TNC/Data/Statewide/LUCAS/scripts/Demonstration
> Library.ssim.input/Scenario-1/STSim_InitialConditionsSpatial/\Library\Frameworks\R.framework\Versions\3.3\Resources\library\rsyncrosim\extdata\initial-stratum.tif

I've tried moving the raster files out of the package folder and into my working directory, and get a similar error:

> Error in tt[[1]] : subscript out of bounds
The specified external file does not exist:
> /Users/davidmarvin/Documents/TNC/TNC/Data/Statewide/LUCAS/scripts/Demonstration Library.ssim.input/Scenario-1/STSim_InitialConditionsSpatial/\Users\davidmarvin\Documents\TNC\TNC\Data\Statewide\LUCAS\scripts\initial-stratum.tif

I created a workaround by modifying the saveDatasheet function by commenting out line 249 and adding the following code on line 262:

`cDat = as.data.frame(t(sapply(cDat,function(x) basename(as.character(x)))))`

Basically forcing the saveDatasheet function to remove the absolute path and use just the file name.

**Comment by alexembrey** [comment]
Thanks very much for reporting this bug.  It will be fixed in the next release.

Cheers,
The SyncroSim Development Team

**Comment by LeonardoFrid** [comment]
This appears to have been fixed at some point. This use case was tested on Ubuntu 17.04 (zesty) and there were no problems.
